apps/web/tsconfig.json

```diff
@@ -14,7 +14,11 @@
     "allowJs": true,
     "skipLibCheck": true,
     "forceConsistentCasingInFileNames": true,
-    "noEmit": true
+    "noEmit": true,
+    "baseUrl": ".",
+    "paths": {
+      "@/*": ["./app/*"]
```

Comment:
This will override all paths in our workspace.
We should move components to a lib later.

Reply:
On the other hand this will make it easier to copy existing solution without too much overhead.
Extracting component could be for the future.